main.go

```diff
@@ -106,6 +106,8 @@ func main() {
 	daemonCommand.PersistentFlags().String(f.Kubernetes.TLS.KeyFile, "", "Key file path to use to authenticate with Kubernetes.")
 	daemonCommand.PersistentFlags().String(f.Vault.Address, "", "Address used to connect to Vault.")
 	daemonCommand.PersistentFlags().String(f.Vault.Token, "", "Token used to authenticate against Vault.")
+	daemonCommand.PersistentFlags().String(f.Vault.PKI.CATTL, "", "TTL used to generate a new Cluster CA.")
+	daemonCommand.PersistentFlags().String(f.Vault.PKI.CommonNameFormat, "", "Common name used to generate a new Cluster CA.")
```

Comment:
Super picky and not really important in this specific case but I want to share my thoughts on this topic now because it will become quite important soon. When we deal with configuration I would like to think twice on the structure. I made the experience over the last years that structuring configuration is super important to understand and maintain it. So I think it should be `f.Vault.PKI.CA.TTL` and `f.Vault.PKI.CommonName.Format`. Just as a side note, this goes towards the story of configuration of G8S installations in general and how we deal with this. See also https://github.com/giantswarm/giantswarm/issues/1030.